suites/squid/common/regression/nvme-single-site-deploy-and-configure.yaml

```diff
@@ -0,0 +1,172 @@
+                - cephfs
+          - config:
+              command: shell
+              args:
+                - "ceph osd pool create rbd"
+          - config:
+              command: shell
+              args:
+                - "rbd pool init rbd"
+      abort-on-fail: true
+
+  - test:
+      name: Configure client
+      desc: Configure the RGW & RBD clients
+      module: test_client.py
+      config:
+        command: add
+        id: client.1
+        node: node18
+        install_packages:
+          - ceph-common
+          - rbd-nbd
+          - jq
+          - fio
+        copy_admin_keyring: true
+        caps:
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      abort-on-fail: true
+
+  - test:
+      name: Configure client
+      desc: Configure the RGW & RBD clients
+      module: test_client.py
+      config:
+        command: add
+        id: client.2
+        node: node19
+        install_packages:
+          - ceph-common
+          - rbd-nbd
+          - jq
+          - fio
+        copy_admin_keyring: true
+        caps:
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      abort-on-fail: true
+
+  - test:
+      name: Configure client
+      desc: Configure the RGW & RBD clients
+      module: test_client.py
+      config:
+        command: add
+        id: client.3
+        node: node20
+        install_packages:
+          - ceph-common
+          - rbd-nbd
+          - jq
+          - fio
+        copy_admin_keyring: true
+        caps:
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      abort-on-fail: true
+
+  - test:
+      name: Configure client
+      desc: Configure the RGW & RBD clients
+      module: test_client.py
+      config:
+        command: add
+        id: client.4
+        node: node21
+        install_packages:
+          - ceph-common
+          - rbd-nbd
+          - jq
+          - fio
+        copy_admin_keyring: true
+        caps:
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      abort-on-fail: true
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: shell
+              args:
+                - ceph osd pool create rbd
+          - config:
+              command: shell
+              args:
+                - ceph osd pool application enable rbd rbd
+          - config:
+              command: shell
+              args:
+                - rbd create -s 1G image1
+          - config:
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  label: nvmeof-gw
+              pos_args:
+                - rbd
+                - gw_group1
+      desc: NVMeoF Gateway deployment using cephadm
```

Comment:
As per last discussion, don't this step will be taken care in NvmeOf test suites ?